### PR TITLE
Support fetch test case in parser

### PR DESCRIPTION
**Pull request by darionyaphet**

### CODE REVIEW

File: src/parser/test/ParserTest.cpp [hunk 1]
```diff
@@ -715,6 +715,133 @@ TEST(Parser, DeleteEdge) {
...
+        GQLParser parser;
+        std::string query = "$var = GO FROM 1 over e1; "
+                            "FETCH PROP ON person $var.id YIELD person.name, person.age";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "FETCH PROP ON player 1,2,3 "
+                            "YIELD DISTINCT person.name, person.age";
```
Comment (CPWstatic) [inline src/parser/test/ParserTest.cpp]: There is no syntax error, but the semantics are wrong. I suggest change the tag of person to player.

File: src/parser/test/ParserTest.cpp [hunk 2]
```diff
@@ -715,6 +715,133 @@ TEST(Parser, DeleteEdge) {
...
+                            "YIELD transfer._src AS s, serve._dst AS d | "
+                            "FETCH PROP ON transfer $-.s -> $-.d YIELD transfer.amount";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "$var = GO FROM 12345 OVER transfer "
+                            "YIELD transfer._src AS s, edu._dst AS d; "
+                            "FETCH PROP ON serve $var.s -> $var.d YIELD transfer.amount";
```
Comment (CPWstatic) [inline src/parser/test/ParserTest.cpp]: Same problem as mentioned above.

Comment (darionyaphet) [reply src/parser/test/ParserTest.cpp]: This case is refer to [fetch-syntax](https://github.com/vesoft-inc/nebula/blob/master/docs/manual-EN/statement-syntax/data-manipulation-statements/fetch-syntax.md).  But no mistakes were found.

File: src/parser/test/ParserTest.cpp (hunk 1 above)
Comment (darionyaphet) [reply src/parser/test/ParserTest.cpp]: Nice

File: src/parser/test/ParserTest.cpp (hunk 2 above)
Comment (CPWstatic) [reply src/parser/test/ParserTest.cpp]: I mean you are about to fetch prop on serve while yield from transfer.

**Pull request merged**